Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json

```diff
@@ -1,5069 +1,5230 @@
+        ],
+        "version": "1.1.0",
+        "title": "BETTER Mobile Threat Defense (MTD)",
+        "templateRelativePath": "BETTER_MTD_Workbook.json",
+        "subtitle": "",
+        "provider": "BETTER Mobile"
+    },
+    {
+        "workbookKey": "AlsidIoEWorkbook",
+        "logoFileName": "Alsid.svg",
+        "description": "Workbook showcasing the state and evolution of your Alsid for AD Indicators of Exposures alerts.",
+        "dataTypesDependencies": [
+            "AlsidForADLog_CL"
+        ],
+        "dataConnectorsDependencies": [
+            "AlsidForAD"
+        ],
+        "previewImagesFileNames": [
+            "AlsidIoEBlack1.png",
+            "AlsidIoEBlack2.png",
+            "AlsidIoEBlack3.png",
+            "AlsidIoEWhite1.png",
+            "AlsidIoEWhite2.png",
+            "AlsidIoEWhite3.png"
+        ],
+        "version": "1.0.0",
+        "title": "Alsid for AD | Indicators of Exposure",
+        "templateRelativePath": "AlsidIoE.json",
+        "subtitle": "",
+        "provider": "Alsid"
+    },
+    {
+        "workbookKey": "AlsidIoAWorkbook",
+        "logoFileName": "Alsid.svg",
+        "description": "Workbook showcasing the state and evolution of your Alsid for AD Indicators of Attack alerts.",
+        "dataTypesDependencies": [
+            "AlsidForADLog_CL"
+        ],
+        "dataConnectorsDependencies": [
+            "AlsidForAD"
+        ],
+        "previewImagesFileNames": [
+            "AlsidIoABlack1.png",
+            "AlsidIoABlack2.png",
+            "AlsidIoABlack3.png",
+            "AlsidIoAWhite1.png",
+            "AlsidIoAWhite2.png",
+            "AlsidIoAWhite3.png"
+        ],
+        "version": "1.0.0",
+        "title": "Alsid for AD | Indicators of Attack",
+        "templateRelativePath": "AlsidIoA.json",
+        "subtitle": "",
+        "provider": "Alsid"
+    },
+    {
+        "workbookKey": "InvestigationInsightsWorkbook",
+        "logoFileName": "Microsoft_logo.svg",
+        "description": "Help analysts gain insight into incident, bookmark and entity data through the Investigation Insights Workbook. This workbook provides common queries and detailed visualizations to help an analyst investigate suspicious activities quickly with an easy to use interface. Analysts can start their investigation from a Microsoft Sentinel incident, bookmark, or by simply entering the entity data into the workbook manually.",
+        "dataTypesDependencies": [
+            "AuditLogs",
+            "AzureActivity",
+            "CommonSecurityLog",
+            "OfficeActivity",
+            "SecurityEvent",
+            "SigninLogs",
+            "ThreatIntelligenceIndicator"
+        ],
+        "dataConnectorsDependencies": [
+            "AzureActivity",
+            "SecurityEvents",
+            "Office365",
+            "AzureActiveDirectory",
+            "ThreatIntelligence",
+            "ThreatIntelligenceTaxii",
+
```

Comment:
please remove ' , ' after the curly brace

Reply:
Hi, all done now.

Reply:
for playbbooks
AAD Revoke SignIn session - metadata is added
For Isolate and Unisolate MDE Machine - entity trigger - metadata is filled properly. I normally don't delete empty values but leave them if something needs to be added later.